doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,392 @@
+
+.. code-block:: text
+
+    subsection insertion info
+    	set insertion method					= non_uniform
+    	set inserted number of particles at each time step  	= 2000
+    	set insertion frequency            		 	= 2000
+    	set insertion box minimum x            	 		= -0.012
+    	set insertion box minimum y            	        	= 0
+    	set insertion box minimum z            	        	= -0.012
+    	set insertion box maximum x            	        	= 0.012
+    	set insertion box maximum y           	 		= 0.1
+    	set insertion box maximum z            	        	= 0.012
+    	set insertion distance threshold			= 1.8
+    	set insertion random number range			= 0.3
+    	set insertion random number seed			= 19
+    end
+
+
+Floating walls
+~~~~~~~~~~~~~~~~~~~
+
+We need to pack the particles in the middle of the cylinder. Therefore, we create a stopper (floating wall) somewhere below the center of the bed. We chose the point with a y-coordinate of -0.06 to create the wall. We then define a normal to the wall at this point. Make sure that the end time of the floating wall is bigger than the simulation time to ensure that the particles remain suspended. This is shown in:
+
+.. code-block:: text
+
+    subsection floating walls
+      set number of floating walls	                = 1
+    	subsection wall 0
+		subsection point on wall
+			set x				= 0
+			set y				= -0.06
+			set z				= 0
+		end
+		subsection normal vector
+			set nx				= 0	
+			set ny				= 1
+ 			set nz 				= 0
+		end
+		set start time				= 0
+		set end time				= 5
+   	end
+
+    end
+    
+Running the simulation
+----------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``dem_3d`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+
+  dem_3d packing_in_circle.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+
+  mpirun -np 8 dem_3d packing_in_circle.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+
+.. note:: 
+    The vtu files generated by Lethe are compressed archives. Consequently, they cannot be postprocessed directly. Although they can be easily post-processed using Paraview, it is sometimes necessary to be able to work with the raw data. The python library `PyVista <https://www.pyvista.org/>`_  allows us to do this. 
```

Comment:
```suggestion



```
This is already elsewhere so I don't think this is necessary in a more advanced tutorial like this one?